addon/utils/calculate-position.js

```diff
@@ -24,14 +25,22 @@ export default function(_, _2, { renderInPlace }) {
   }
 }
 
-export function calculateWormholedPosition(trigger, content, { horizontalPosition, verticalPosition, matchTriggerWidth, previousHorizontalPosition, previousVerticalPosition }) {
+export function calculateWormholedPosition(trigger, content, destination, { horizontalPosition, verticalPosition, matchTriggerWidth, previousHorizontalPosition, previousVerticalPosition }) {
   // Collect information about all the involved DOM elements
   let scroll = { left: window.pageXOffset, top: window.pageYOffset };
   let { left: triggerLeft, top: triggerTop, width: triggerWidth, height: triggerHeight } = trigger.getBoundingClientRect();
   let { height: dropdownHeight, width: dropdownWidth } = content.getBoundingClientRect();
   let viewportWidth = self.document.body.clientWidth || self.window.innerWidth;
   let style = {};
 
+  // Apply containers' offset
+  let destinationParentPosition = window.getComputedStyle(destination.parentNode).position;
```

Comment:
Maybe a `getPositionParent` could be called? So that the dev doesn't have to worry about which node they are adding the dropdown to and what the styling is, but the dropdown will automatically position itself correctly no matter where it is.